### ISSUE TITLE
Install on debian 12 

### ISSUE THREAD

**Issue by Aurel-Charles** [pibooth/pibooth#586]
**Describe the bug**

got an error when installing pibooth

pi@raspberrypi:~ $ sudo pip3 install pibooth[dslr,printer]
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    For more information visit http://rptl.io/venv

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.

**Environment**
pi@raspberrypi:~ $ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 12 (bookworm)
Release:	12
Codename:	bookworm

Im not really familiar with all of this so a little help would could much apprciated 😉

**Comment by t-animal** [comment]
I came across the same problem and installed pibooth to a virtualenv:

```
mkdir ~/pibooth
cd ~/pibooth
python -m venv venv
source venv/bin/activate
pip3 install pibooth[dslr,printer]
```

Installing system-wide packages using pip is generally considered bad practice. However, since then I didn't get pibooth to run, it throws exceptions left and right. So I'm not sure if running it in a virtualenv is even supported?

**Comment by t-animal** [comment]
Ah, I see. https://github.com/pibooth/pibooth/issues/540 I guess this issue should be closed in favor of #540.

**Comment by werdeil** [comment]
Indeed, I'll close this as #540 is used to follow up everything.

Debian 12 and all Python 3.11+ OS are indeed introducing a change on the way Python projects are installed by encouraging the use of virtualenv. We will need to update the install instruction to add something about it (either use virtualenv or the `--break-system-packages` option)
